### DISCUSSION TITLE
Can I specify in quarto configuration to restrict image file of less than certain size

### DISCUSSION THREAD

**Discussion by Pratichhya** [quarto-dev/quarto-cli#9273] [Q&A]
### Description

I'm not sure where it goes, maybe in _quarto.yml or maybe even within the yml part of qmd. 

---
max_image_size = 1MB
---

Is there any such feature available in quarto?

**Comment by mcanouil** [comment]
No there is not.

You could make a pre-render script for that purpose, see <https://quarto.org/docs/projects/scripts.html#pre-and-post-render>.
